### ISSUE TITLE
integrate offshore BOS model

### ISSUE THREAD

**Issue by dykesk** [WISDEM/Plant_CostsSE#3]
Figure out status of offshore BOS model that Mike Maness coded up and get it integrated to plant_costsSE and then update WISDEM LCOE assemblies to use offshore BOS model

**Comment by kilojoules** [comment]
Should I email Mike? Where can I find the offshore BOS model?

**Comment by dykesk** [comment]
yeah - i'm sure its on the ydrive somewhere, but i'm not sure where, i
would email Mike to see what the status is and the location
